### ISSUE TITLE
add qexp()

### ISSUE THREAD

**Issue by MaximeLemonnier-Leddartech** [matthew-brett/transforms3d#21]
````
def qexp(q):
    ''' Return exponential of quaternion
    Parameters
    ----------
    q : 4 element sequence
       w, i, j, k of quaternion

    Returns
    -------
    exp(q) : the quaternion exponential
    '''
    w, v = q[0], q[1:]
    norm = np.sqrt(np.dot(v, v))
    result = np.zeros((4,), norm.dtype)
    result[0] =  np.cos(norm)
    result[1:] = np.sin(norm)/norm * v
    return result * np.exp(w)
```


very useful to integrate angular velocity, see e.g.: https://www.ashwinnarayan.com/post/how-to-integrate-quaternions/

**Comment by matthew-brett** [comment]
I'm really sorry to be slow to respond to this.  Would you consider making a pull request, and adding a test?

**Comment by MaximeLemonnier-Leddartech** [comment]
Just did, sorry for the delay...

**Comment by MaximeLemonnier-Leddartech** [comment]
Hi Mathhew,

If you remember, you merge my branch adding exp() and log(), any chances you publish that on pip?

**Comment by matthew-brett** [comment]
Closed by #24
